### PR TITLE
MAN-213 track versions of entities

### PR DESCRIPTION
**Pull request by nomadicoder**

### CODE REVIEW

**Comment by nomadicoder** [comment]
Needs test for Whodunit

**Comment by nomadicoder** [comment]
@bibliotechy @cdoyle-temple Ready for review.

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -5,11 +5,7 @@ class ApplicationController < ActionController::Base
   # user, we have no access to the devise object. So, we need to override
   # current_user to return the current account. This is needed both
   # in ApplicationController and in Admin::ApplicationController
-  before_action :get_alert, :set_footer
-
-  def current_user
-    current_account
-  end
+  before_action :get_alert, :set_footer, :set_paper_trail_whodunnit
```
Comment (bibliotechy) [inline app/controllers/application_controller.rb]: Aren't get alert and set footer just used for display? Do they need whodunnits?

File: db/schema.rb [hunk 2]
```diff
@@ -199,8 +199,6 @@
   create_table "groups", force: :cascade do |t|
     t.string "name"
     t.text "description"
-    t.string "phone_number"
```
Comment (bibliotechy) [inline db/schema.rb]: @skng5  these keep coming up in your PRs. Why?

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (nomadicoder) [reply app/controllers/application_controller.rb]: @bibliotechy According to the documentation, this is where it's supposed to go

File: db/schema.rb (hunk 2 above)
Comment (nomadicoder) [reply db/schema.rb]: @bibliotechy I may have to start with a fresh working repo.

**Comment by nomadicoder** [comment]
@bibliotechy please review.

**Pull request merged**